generators/app/index.js

```diff
@@ -1,114 +1,109 @@
-var generators = require('yeoman-generator');
-var _ = require('lodash');
+var generators = require('yeoman-generator')
+var _ = require('lodash')
 
 module.exports = generators.Base.extend({
 
   _paths: function () {
-    this.log( this.destinationRoot() );
-    this.log( this.destinationPath( this.classname + '.js') );
-    this.log( this.sourceRoot() );
-    this.log( this.templatePath('index.js') );
+    this.log(this.destinationRoot())
+    this.log(this.destinationPath(this.classname + '.js'))
+    this.log(this.sourceRoot())
+    this.log(this.templatePath('index.js'))
   },
 
   // The name `constructor` is important here
```

Comment:
Why? Add more detail. Comments are not only for you to read the code better later but for the entire team. Please see JSDoc format to add comments.